specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/workspaceRP.json

```diff
@@ -2959,10 +3020,11 @@
       "x-ms-discriminator-value": "ServicePrincipal"
     },
     "ServiceTagDestination": {
```

Comment:
>ServiceTagDestination [](http://example.com/codeflow?start=5&length=21)

this looks a lot like something we should ideally refactor in common-types later also, what do you think?

Reply:
https://msdata.visualstudio.com/Vienna/_workitems/edit/2688199